From b50754aa4bf3cb0cbb5983b5f8b1d1cf54e8e5c9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Christian=20P=C3=B6tzsch?=
 <christian.poetzsch@kernkonzept.com>
Date: Fri, 6 Sep 2024 10:40:08 +0200
Subject: [PATCH 1/2] Disable tests subdir

They make trouble when configure runs in bob. Simply disable them.
---
 meson.build | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/meson.build b/meson.build
--- a/meson.build
+++ b/meson.build
@@ -3881,8 +3881,8 @@
 subdir('linux-user')
 
 # needed for fuzzing binaries
-subdir('tests/qtest/libqos')
-subdir('tests/qtest/fuzz')
+#subdir('tests/qtest/libqos')
+#subdir('tests/qtest/fuzz')
 
 ##############################################
 # Internal static_libraries and dependencies #
@@ -4610,9 +4610,9 @@
 subdir('pyvenv')
 # Tests are disabled on emscripten because they rely on host features that aren't
 # supported by emscripten (e.g. fork and unix socket).
-if host_os != 'emscripten'
-  subdir('tests')
-endif
+#if host_os != 'emscripten'
+#  subdir('tests')
+#endif
 if gtk.found()
   subdir('po')
 endif
-- 
2.47.3
